hardware/ThermostatHardware.h

```diff
@@ -0,0 +1,48 @@
+#pragma once
+
+#include <iostream>
+#include "DomoticzHardware.h"
+
+class CThermostatHardware : public CDomoticzHardwareBase
+{
+public:
+
+	//thermostat available mode in string "OFF,ECO,COMFORT,AUTO," separated by comma, 
+	// and in the order from 0 to N and 0 = OFF 1 = ECO, 2 = CONFOR 3 = AUTO
+	std::string AvailableMode;
```

Comment:
Could you make member variables start with 'm_'
I think this variable and other functions below should be plural (m_AvailableMode, GetAvailableModes, ...)

Reply:
ok !done